src/main/scala/introprog/ImageFilter.scala

```diff
@@ -0,0 +1,82 @@
+
+
+package introprog
+
+
+/**
+ * Superklassen till alla filterklasser.
+ * 
+ * @version 1.3 (2021-07-25) översättning från java
```

Comment:
Unsure if this should be a version number here as it creates confusion with the introporg version number. 
So remove version numbers but keep dates and authors.